compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -114,89 +114,6 @@ enumerateMatmulTileArm64(TypeRange elementTypes, ExecutableTargetAttr target) {
     }
   }
 
-  if (hasUkernel(target) && lhs.isSignlessInteger(8) &&
```

Comment:
Nooo! I was saying replace `hasUkernel` by `true`, so the `if(hasUkernel)` branches are unconditionally taken and the `if(!hasUkernel)` branches are unconditionally not taken :-) @lialan

Reply:
duh... what was I thinking...